src/Table/Body.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+
+import * as React from "react";
+import { colors } from "../constants";
+import Row from "./Row";
+import { isOdd, getChidlName } from "./helpers";
+
+type Props = {
+  children: React.Element<typeof Row>,
+};
+
+class Body extends React.Component<Props> {
+  static Row = Row;
+
+  getBackgroundColor = (index: number) => ({
+    default: isOdd(index) ? colors.white.normal : colors.cloud.light,
+    hover: colors.cloud.normal,
+  });
+
+  render() {
+    const children = React.Children.map(this.props.children, (child, index) => {
+      if (child.type !== Row)
+        console.error(`Invalid children type ${getChidlName(child)}, it should be Row component.`); // eslint-disable-line no-console
+
+      return React.cloneElement(child, {
+        isOdd: isOdd(index),
+        backgroundColor: this.getBackgroundColor(index),
+        style: { borderBottom: `1px ${colors.cloud.normal} solid` },
```

Comment:
why this style can't be defined in Row directly?

Reply:
Because I used same `Row` Component in `Footer` and `Header` and, there I don't need this styles there.
These styles are just for `Rows` inside `Body`, and this looks to me like a better approach then define everything inside. But I thinking about improvements.